practice/src/scripts/models/task.model.ts

```diff
@@ -70,6 +82,106 @@ class TaskModel {
             storage.saveTasks(this.tasks);
         }
     };
+
+
+    toggleTask = (
+        id: number,
+        type: string,
+        renderTasks: (tasks: Task[]) => void
+    ) => {
+        const currentTime = new Date();
+        const hours = currentTime.getHours();
+        const formattedHours = hours % 24 || 24;
+        const minutes = currentTime.getMinutes();
+        const seconds = currentTime.getSeconds();
+        const formattedTime = `${formattedHours}:${minutes}:${seconds
+            .toString()
+            .padStart(2, "0")}, ${currentTime.toLocaleDateString("en-US", {
+            year: "numeric",
+            month: "short",
+            day: "numeric",
+        })}`;
+
+        if (type === "active") {
```

Comment:
don't use alot of if - else in a function
should use switch - case
and split function